internal/adapters/mongodb/connection.go

```diff
@@ -0,0 +1,30 @@
+package mongodb
```

Comment:
This could be inside the configuration folder, since it's a MongoDB connection configuration